cPP_DSC.adoc

```diff
@@ -312,21 +312,42 @@ The following paragraphs describe the relationships between elements of the DSC.
 
-The administrator may, among other things, accept responsibility for providing timely updates to the DSC, both feature updates and security updates. It may also be responsible for managing the pre-installed SDOs and the initial configuration of the DSC. Different administrators may have different authorities to manage the TSF; for example, one administrator may be responsible for controlling firmware updates while another may take an active part in managing the contents of the DSC installed post-manufacture.
+* ADM-R - Owner Admin role - the administrator role related to the management of the DSC once it has been integrated into a platform.
+* MFGADM-R - Manufacturer Admin role - the administrator role related to the management of firmware and key material that form the basis for the root of trust.
+* CApp-R - Client Application role - the client role of the DSC that requests and utilizes the functionality provided by the DSC.
 
-An administrator may manage the contents of the DSC, including user content. A DSC administrator is not necessarily the owner of a given SDO. Although the DSC administrator may possibly own one or more SDOs, not all SDOs allow a DSC administrator direct control of it. In some cases, a DSC administrator may also be in a position to grant or deny another administrator access to what it perceives as their content, namely the DSC's firmware and possibly some keying material belonging to the manufacturer. A DSC manufacturer's choice of allowing an administrator of the DSC this kind of latitude is a feature of its product.
+Depending on the configuration of the DSC, there may not be a separation of the Admin roles, such that the capabilities of the ADM-R and MFGADM-R roles are combined into a single role. For the purposes of the cPP, unless specifically called out, all administration roles are assumed to be combined and will use ADM-R.
 
-The CA role may also be further divided into multiple users. CAs can include:
+As the DSC is generally a component within a larger system or platform, the roles of the DSC are specific to the DSC. While there may be matching roles between the DSC and its platform, the roles here are specifically those in the DSC and are independent of any defined on the platform.
+
+The ADM-R role provides sufficient privileges to manage the functionality of the DSC. As a role designed for the administrator, this role may be responsible for the following:
+
+* Manage access control for SDOs (does not mean the contents of any particular SDO can be read by the administrator)
+* Manage the initial configuration of the DSC
+
+The MFGADM-R role, if explicitly defined in a DSC, may include the following responsibilities (which may be part of the ADM-R role otherwise):
+
+* Manage the pre-installed SDOs and initial configuration of the DSC
```

Comment:
why only initial configuration? Shouldn't we say "... and runtime configuration, if applicable ..."?

Reply:
The way I had interpreted the descriptions initially seemed to focus more on the initial configuration, so I don't know. I don't think it matters too much and have no problem adding that, but I don't know if we really are expecting the manufacturer to control the runtime configuration. Easy to add though if we want it.